doc/functions.dx

```diff
@@ -242,23 +242,16 @@ them in parens like `def`.
 map (\(a:Float). a + 1) [1., 2., 3.]
 > [2., 3., 4.]
 
-'Like `def`, we can also specify implicit parameters with curly braces,
+'Also like `def`, we can specify implicit parameters with curly braces:
 
 \{a} (x:a). x
 > \a:Type ?-> \x:a. x
```

Comment:
Should we adjust the pretty printer to include those parens? This type looks like it wouldn't parse with your changes.

Reply:
Come to think of it, that was printing the function (not its type), and it looks like it wouldn't have parsed before, either.  Anyway, changed the pretty-print of lambda terms to emit the new syntax.